Riot/Modules/Room/Views/BubbleCells/VoiceMessage/VoiceMessageBubbleCell.swift

```diff
@@ -0,0 +1,53 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+class VoiceMessageBubbleCell: SizableBaseBubbleCell, BubbleCellReactionsDisplayable {
+    
+    private var playbackController: VoiceMessagePlaybackController!
+    
+    override func render(_ cellData: MXKCellData!) {
+        super.render(cellData)
+        
+        guard let data = cellData as? RoomBubbleCellData else {
+            return
+        }
+        
+        guard data.attachment.type == MXKAttachmentTypeVoiceMessage else {
+            fatalError("Invalid attachment type passed to a voice message cell.")
+        }
+        
+        playbackController.attachment = data.attachment
+    }
+    
+    override func setupViews() {
+        super.setupViews()
+        
+        bubbleCellContentView?.showSenderInfo = true
+        bubbleCellContentView?.showPaginationTitle = false
+        
+        guard let contentView = bubbleCellContentView?.innerContentView else {
+            return
+        }
+        
+        playbackController = VoiceMessagePlaybackController(mediaServiceProvider: VoiceMessageMediaServiceProvider.sharedProvider,
+                                                            cacheManager: VoiceMessageAttachmentCacheManager.sharedManager)
+        bubbleCellContentView?.addSubview(playbackController.playbackView)
```

Comment:
This line (49) seems redundant to me, we're already adding that view on `contentView`.